paltas/Configs/config_handler.py

```diff
@@ -685,7 +689,10 @@ def draw_image(self,new_sample=True):
 			r = util.array2image(np.sqrt(x_grid**2+y_grid**2))
 			image[r<=self.config_module.mask_radius] = 0
 
-		metadata['seed'] = seed
+		# Write seed to metadata if the image generation passed cuts.
+		if metadata is not None:
```

Comment:
Instead of having a second path through the code where image and metadata are None, it might be clearer to raise a custom exception and catch it in generate.py when a magnification cut fails.

Reply:
Good idea. Implemented and made sure error was verbose.